### PR TITLE
Revise PSP deprecation blog article to note that PSP is now removed

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #36199

**Note**: Once the changes get approved, the localization team has to perform the task to update the blog in their respective language as well.

### CODE REVIEW

**Comment by divya-mohan0209** [comment]
I would use the Disclaimer shortcode here. However since @sftim has proposed the change, I'd like him to opine on this as well.

**Comment by sftim** [comment]
> I would use the Disclaimer shortcode here. However since @sftim has proposed the change, I'd like him to opine on this as well.

I only know of a disclaimer shortcode for "this documentation page mentions third party software". I don't think we need to add that to this article.

File: content/en/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ title: "PodSecurityPolicy Deprecation: Past, Present, and Future"
 date: 2021-04-06
 slug: podsecuritypolicy-deprecation-past-present-and-future
 ---
+***Update 2022/08/23:***
+*With the release of Kubernetes v1.25, PodSecurityPolicy has been removed. This now-historical blog post documents the beginning of the removal process.*
+
+*The release blog has more information about the removal of PSP [here](https://kubernetes.io/blog/2022/08/23/kubernetes-v1-25-release/#pod-security-changes).*
```
Comment (sftim) [inline content/en/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md:10]: Thanks, but this should go _after_ the author information.

File: content/en/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ title: "PodSecurityPolicy Deprecation: Past, Present, and Future"
 date: 2021-04-06
 slug: podsecuritypolicy-deprecation-past-present-and-future
 ---
+***Update 2022/08/23:***
```
Comment (sftim) [inline content/en/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md:7]:
```suggestion
**Update**
```

**Review by sftim — COMMENTED** [review]
The shortcode you could use for the notice is [`pageinfo`](https://www.docsy.dev/docs/adding-content/shortcodes/#pageinfo)

**Comment by sftim** [comment]
/retitle Revise PSP deprecation blog article to note that PSP is now removed

File: content/en/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md (hunk 1 above)
Comment (NitishKumar06) [reply content/en/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md:10]: Before the author's name would make a difference?

Comment (NitishKumar06) [reply content/en/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md]: DONE!

[the remark this answers is not in the document]
Comment (NitishKumar06) [reply content/en/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md]: DONE!

File: content/en/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md (hunk 2 above)
Comment (NitishKumar06) [reply content/en/blog/_posts/2021-04-06-PodSecurityPolicy-Past-Present-and-Future.md]: DONE!

**Review by sftim — COMMENTED** [review]
Thanks
/lgtm
/approve

**Pull request merged**